acme/api/internal/sender/useragent.go

```diff
@@ -1,7 +1,6 @@
 package sender
 
-// CODE GENERATED AUTOMATICALLY
-// THIS FILE MUST NOT BE EDITED BY HAND
+// Code generated by 'internal/useragent'; DO NOT EDIT.
```

Comment:
> This line must appear before the first non-comment, non-blank text in the file.

Reply:
(The remark above ought to be part of the review. It becomes obsolete after changing the templates and regenerating the files.)